internal/services/appservice/linux_function_app_resource.go

```diff
@@ -1199,16 +1203,16 @@ func (r LinuxFunctionAppResource) CustomImporter() sdk.ResourceRunFunc {
 		if props.ServerFarmID == nil {
 			return fmt.Errorf("determining Service Plan ID for Linux %s: %+v", id, err)
 		}
-		servicePlanId, err := parse.ServicePlanID(*props.ServerFarmID)
+		servicePlanId, err := commonids.ParseAppServicePlanIDInsensitively(*props.ServerFarmID)
```

Comment:
just double checking it was intentional to use `ParseAppServicePlanIDInsensitively` given it wasn't case insensitive before?

Reply:
Yeah, unfortunately the API always returns this in lowercase currently, so any processing of a response must be case insensitive.